include/mbgl/ios/MGLTypes.h

```diff
@@ -12,3 +12,8 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode)
     /** The map follows the user location and rotates when the heading changes. */
     MGLUserTrackingModeFollowWithHeading
 };
+
+typedef struct {
+    CLLocationCoordinate2D sw;
+    CLLocationCoordinate2D ne;
+} MGLMapBounds;
```

Comment:
“Bounds” has a specific meaning in Cocoa that we should avoid overloading in the public interface. `MGLMapView.bounds` can’t be set to an `MGLMapBounds` for instance.